media-fonts/ibm_plex/ibm_plex-0.5.3.recipe

```diff
@@ -0,0 +1,37 @@
+SUMMARY="The package of IBM’s new typeface, IBM Plex"
+DESCRIPTION="\
+Thanks for your interest in IBM Plex, IBM’s new corporate type family. \
+We welcome your feedback and contributions to this project. \
+IBM Plex is open source, and should be used by IBMers for all typographical situations, \
+replacing Helvetica Neue, whenever possible\
+"
+HOMEPAGE="https://ibm.github.io/type"
+SOURCE_URI="https://github.com/IBM/type/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="0e212de6d0f33604b6c22e9743aab05bc12d482b1bdc7402972f3f6a9aa92a77"
+LICENSE="SIL Open Font License v1.1"
+COPYRIGHT="2017 IBM Corp. with Reserved Font Name 'Plex'"
+REVISION="1"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE=yes
+
+PROVIDES="ibm_plex = $portVersion"
+REQUIRES=""
+BUILD_REQUIRES=""
+BUILD_PREREQUIRES="coreutils"
+
+SOURCE_DIR="type-$portVersion"
```

Comment:
Could you please move `SOURCE_DIR` just after `CHECKSUM_SHA256`?
Could you please also sort these variables using the recommended order?
According to the [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering), the prefered order (in that part of the recipe) is:
```
HOMEPAGE
COPYRIGHT
LICENSE
REVISION
SOURCE_URI
CHECKSUM_SHA256
SOURCE_DIR
```

Thanks!

Hmm, the style for `DESCRIPTION` has changed some time ago.
Here's an [example using the recommended style](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#a-simple-example-using-the-recommended-style-for-description).

So, for this recipe, we would use:
```
DESCRIPTION="Thanks for your interest in IBM Plex, IBM’s new corporate type \
family. We welcome your feedback and contributions to this project. IBM Plex \
is open source, and should be used by IBMers for all typographical situations, \
replacing Helvetica Neue, whenever possible."
```
But maybe it's OK to keep it the style in your current commit.